index.js

```diff
@@ -1047,33 +1014,6 @@ class ClientServiceBase {
 
     return this.signer;
   }
-
-  /**
-   * @param {LedgerValidationResult} ledgerResult
-   * @param {LedgerValidationResult} auditorResult
-   * @return {LedgerValidationResult}
-   */
-  _validateResult(ledgerResult, auditorResult) {
-    if (this._isAuditorEnabled()) {
-      let code = StatusCode.INCONSISTENT_STATES;
-      if (
-        ledgerResult.getCode() === StatusCode.OK &&
-        auditorResult.getCode() === StatusCode.OK &&
-        ledgerResult.getProof() !== null &&
-        auditorResult.getProof() !== null &&
-        ledgerResult.getProof().hashEquals(auditorResult.getProof().getHash())
-      ) {
-        code = StatusCode.OK;
-      }
-      return new LedgerValidationResult(
-          code,
-          ledgerResult.getProof(),
-          auditorResult.getProof(),
-      );
-    } else {
-      return ledgerResult;
-    }
-  }
```

Comment:
Deleted due to https://github.com/scalar-labs/scalardl-javascript-sdk-base/pull/85/files#r969210304

We don't need it anymore.